### DISCUSSION TITLE
Object literal may only specify known properties, and 'target' does not exist in type 'NuxtConfig'

### DISCUSSION THREAD

**Discussion by andyjamesn** [nuxt/nuxt#20509] [Questions] [answered]
I upgraded an app from an older version of Nuxt 3 and I am getting a warning in my nuxt.config

I am using `target: 'static'`

`
export default defineNuxtConfig({
  target: 'static',
});
`

I get the following error

_Object literal may only specify known properties, and 'target' does not exist in type 'NuxtConfig'_

<img width="1318" alt="CleanShot 2023-04-25 at 22 23 37@2x" src="https://user-images.githubusercontent.com/54320410/234408729-85f7c67d-b115-49ff-ac3c-b39d7117de8c.png">

**Comment by ndragun92** [comment, marked as answer]
If you want a static build you have to remove the target since SSR: is set to true initially.

Then run the npm run generate command to generate a static build which will be in .output/public folder

There are 3 modes now

SSR: true with npm run build generated Server side rendering build
SSR: false with npm run build generates SPA build
SSR: true with npm run generate creates a static build

Hope it helps
@andyjamesn
